e2e-tests/ukef/cypress/e2e/journeys/portal/test-data/AIN-deal-unissued-facilities/dealReadyToSubmit.js

```diff
@@ -0,0 +1,24 @@
+const dealThatJustNeedsDates = require('./dealThatJustNeedsDates.json');
+const { today } = require('../../../../../../../e2e-fixtures/dateConstants');
+
+module.exports = () => {
```

Comment:
Could this be a named export?


```suggestion
const generateDealWithDates = () => {
};

module.exports = {
  generateDealWithDates,
}
```